src/libmongoc/doc/installing.rst

```diff
@@ -12,7 +12,8 @@ The MongoDB C Driver is continuously tested on a variety of platforms including:
 
 - Archlinux
 - Debian 9.2, 10.0, 11.0
-- macOS 10.14
+- macOS 10.14 (Will be dropped in upcoming release)
```

Comment:
```suggestion
- macOS 10.14 (Will be dropped in an upcoming release)
```

To match the wording in the `NEWS` files.  Otherwise, it can give the impression that support will be dropped in the very next release.

Reply:
Good catch.